hangups/conversation.py

```diff
@@ -11,34 +12,62 @@
 
 
 @asyncio.coroutine
-def build_user_conversation_list(client):
+def build_user_conversation_list(client, conversations_per_request=100):
```

Comment:
Do Google's clients lazy load the conversation list?

I think loading them all is fine for now because hangups currently assumes that it knows about all the conversations.